pkgs/by-name/ti/tickr/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchurl
+
+, fribidi
+, gnutls
+, gtk2
```

Comment:
This PR may not be merged because we are already actively removing gtk2 packages.